doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-configuring-tomcat-security-management-provider-proc.adoc

```diff
@@ -0,0 +1,30 @@
+[id='business-central-configuring-tomcat-security-management-provider-proc']
```

Comment:
We don't support deploying Business central to Tomcat server. I think this part should be removed